packages/web-runtime/src/components/ApplicationsMenu.vue

```diff
@@ -17,18 +18,34 @@
       mode="click"
       :options="{ pos: 'bottom-right', delayHide: 0 }"
       class="uk-width-large"
-      close-on-click
       padding-size="small"
+      close-on-click
     >
-      <ul class="uk-grid-small uk-text-center" uk-grid>
-        <li v-for="(n, nid) in menuItems" :key="`apps-menu-${nid}`" class="uk-width-1-3">
-          <a v-if="n.url" key="apps-menu-external-link" :target="n.target" :href="n.url">
-            <oc-icon :name="n.iconMaterial" size="xlarge" />
-            <span class="uk-display-block" v-text="$gettext(n.title)" />
+      <ul class="oc-p-xs">
+        <li
+          v-for="(n, nid) in menuItems"
+          :key="`apps-menu-${nid}`"
+          style="list-style: none"
+          class="oc-pt-s oc-pb-s"
+        >
+          <a
```

Comment:
cant `router-link` used in both cases?

Reply:
Nicer to be explicit about external links imo